parsimonious/grammar.py

```diff
@@ -296,64 +298,74 @@ def __init__(self, custom_rules=None):
         """
         self.custom_rules = custom_rules or {}
 
-    def visit_parenthesized(self, parenthesized, (left_paren, _1,
-                                                  expression,
-                                                  right_paren, _2)):
+    def visit_parenthesized(self, parenthesized, _a):
         """Treat a parenthesized subexpression as just its contents.
 
         Its position in the tree suffices to maintain its grouping semantics.
 
         """
+        (left_paren, _1, expression, right_paren, _2) = _a
         return expression
 
-    def visit_quantifier(self, quantifier, (symbol, _)):
+    def visit_quantifier(self, quantifier, _a):
         """Turn a quantifier into just its symbol-matching node."""
+        (symbol, _1) = _a
```

Comment:
It's such a shame that Py3 removed support for argument unpacking. I wonder if there's a more readable approach we can take.